week2/project/ecommerce/src/components/CategoryList.jsx

```diff
@@ -0,0 +1,63 @@
+import Category from "./Category";
+
+import "./components-css/category.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function CategoryList(props) {
+  const [categoriesData, setCategoriesData] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    getCategoriesData();
+  }, []);
+
+  async function getCategoriesData() {
+    await axios("https://fakestoreapi.com/products/categories")
+      .then((respose) => {
+        setCategoriesData(respose.data);
+      })
+      .catch((error) => {
+        console.error("Error fetching data: ", error);
```

Comment:
console is for developers you should delete all of them before publishing your code
📝advise📝